sklearn/preprocessing/data.py

```diff
@@ -222,6 +222,10 @@ class MinMaxScaler(BaseEstimator, TransformerMixin):
         Set to False to perform inplace row normalization and avoid a
         copy (if the input is already a numpy array).
 
+    continuous_features : list, optional, default None
```

Comment:
I would not implicitly force this to be applied on continuous features only. You want still want to do that even if features are discrete. I would therefore rename this to `selected` instead.

You could also base the implementation on https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/preprocessing/data.py#L1312